supports/debug.xcodeproj/project.pbxproj

```diff
@@ -114,8 +112,7 @@
 		38A90F89163506F8001DB09E /* debugTests */ = {
 			isa = PBXGroup;
 			children = (
-				38F85958215BB56300CD80AE /* debugTests.m */,
-				38A90F90163506F8001DB09E /* debugCppTests.mm */,
+				38A90F90163506F8001DB09E /* debugTests.mm */,
```

Comment:
there were both `.mm` and `.m` tests to test objetive-c and objective-c++ actually.
Is there any reason to remove `.m` test?

Reply:
sorry, this change was make by the FoundationExtensions repo's cdebug. If that's not need, i would reset the file type  change.